lib/v2/screens/dashboard/interactor/viewmodels/transactions_events.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:equatable/equatable.dart';
+
+abstract class TransactionsEvent extends Equatable {}
+
+class LoadTransactionsEvent extends TransactionsEvent {
```

Comment:
i think we override the to string for this classes, but im not sure.

Reply:
??